react/features/welcome/components/WelcomePage.native.js

```diff
@@ -153,34 +210,23 @@ class WelcomePage extends AbstractWelcomePage {
     }
 
     /**
-     * Renders legal-related content such as Terms of service/use, Privacy
-     * policy, etc.
+     * Enables or disables rendering the local video on the welcome screen.
+     * Also releases the cam if no render is needed.
      *
      * @private
-     * @returns {ReactElement}
+     * @param {boolean} show - To show or not to show the local track.
+     * @returns {void}
      */
-    _renderLegalese() {
-        const { t } = this.props;
+    _setShowVideo(show) {
+        const { dispatch } = this.props;
 
-        return (
-            <View style = { styles.legaleseContainer }>
-                <Link
-                    style = { styles.legaleseItem }
-                    url = { TERMS_URL }>
-                    { t('welcomepage.terms') }
-                </Link>
-                <Link
-                    style = { styles.legaleseItem }
-                    url = { PRIVACY_URL }>
-                    { t('welcomepage.privacy') }
-                </Link>
-                <Link
-                    style = { styles.legaleseItem }
-                    url = { SEND_FEEDBACK_URL }>
-                    { t('welcomepage.sendFeedback') }
-                </Link>
-            </View>
-        );
+        dispatch(setAudioOnly(!show));
```

Comment:
This block doesn't look correct. If there is a video track, we shouldn't destroy it, just by muting it, which setAudioOnly already does should be enough.